src/main/java/piano/SaveAudio/AudioToMidi.java

```diff
@@ -0,0 +1,68 @@
+package piano.SaveAudio;
+
+import piano.recorder.KeyPressedInfo;
+
+import javax.sound.midi.*;
+import java.io.File;
+import java.util.ArrayList;
+
+public class AudioToMidi {
+
+    private ArrayList<KeyPressedInfo> recordedKeysInfo;
+    private File location;
+    private int instrumentNum;
+
+    public AudioToMidi(ArrayList<KeyPressedInfo> getRecordedKeysInfo, File locationToSave) {
+        location = locationToSave;
+        recordedKeysInfo = getRecordedKeysInfo;
+        writeMidiFile();
+    }
+
+    private void writeMidiFile() {
+        try
+        {
+            Sequence sequence = new Sequence(Sequence.PPQ, 4);
+            Sequencer sequencer = MidiSystem.getSequencer();
+            Track currTrack = sequence.createTrack();
+            int numNotes = recordedKeysInfo.size();
+
+            //TODO: fix so takes in proper channel and plays back in that instrument (instead of piano)
+            instrumentNum = 1;
+
+            for (int i = 0; i < numNotes; i++)
+            {
+                int note = recordedKeysInfo.get(i).getLabelPressed().getPitch();
+                // Add Note On event - Command 144 is to play a note.
+                currTrack.add(makeASong(144, instrumentNum, note, 120, i));
+            }
+
+            sequencer.setSequence(sequence);
+            sequencer.setTempoInBPM(120);
```

Comment:
constant numbers in your code are known as "Magic Numbers"
In general you want to have them be named constants so that people understand what they are. 
For instance, you know what 120 and 144 represent, but I don't.